libraries/module-minifier/src/WorkerPoolMinifier.ts

```diff
@@ -167,6 +167,8 @@ export class WorkerPoolMinifier implements IModuleMinifier {
             console.log(`Module minification: ${this._deduped} Deduped, ${this._minified} Processed`);
           }
         }
+        this._deduped = 0;
```

Comment:
Intentionally included in this PR?

Reply:
Yes. Since the `WorkerPoolMinifier` is loaded up only once, we need to reset the values that it is tracking at the end of the execution, otherwise they will just continue incrementing the same values over time.

Reply:
The ModuleMinifierPlugin isn't currently known to be safe to run in watch mode, so that might need fixing separately.